Scaffolder/register-new-component/README.md

```diff
@@ -38,6 +37,10 @@ Repo Host Example:
 
 ## TODO's
 
+### GitHub Proxy
+- [ ] Navigate to `Settings > Developer Settings > Personal access tokens > Tokens (classic)`. Press **Generate new token Classic**.
```

Comment:
Maybe worth mentioning "in GitHub" somewhere on this sentence, and "in Roadie" on the one below

Reply:
Excellent point, updated.